src/ontology/cl-edit.owl

```diff
@@ -21954,6 +21955,19 @@ AnnotationAssertion(rdfs:label obo:CL_0009015 "Peyer's patch follicular dendriti
 EquivalentClasses(obo:CL_0009015 ObjectIntersectionOf(obo:CL_0000442 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001211)))
 SubClassOf(obo:CL_0009015 obo:CL_0000442)
 
+# Class: obo:CL_0009019 (nephrogenic zone cell)
+
+AnnotationAssertion(obo:CL_0009008 obo:CL_0009019 "")
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0009019 "A kidney cortical cell that is part of the nephrogenic zone.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0009019 <http://orcid.org/0000-0002-2825-0621>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0009019 "PMID:29449449")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0009019 "https://orcid.org/0000-0002-3163-0115")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009019 "cortical nephrogenic niche cell")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009019 "cortical nephrogenic zone cell")
+AnnotationAssertion(rdfs:label obo:CL_0009019 "nephrogenic zone cell")
+EquivalentClasses(obo:CL_0009019 ObjectIntersectionOf(obo:CL_0002681 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0009871)))
+SubClassOf(obo:CL_0009019 obo:CL_0002681)
```

Comment:
Delete subClassOf assertion - don't need to duplicate genus also in EquivalentClass assertion.  Not a big deal, but I prefer to avoid.  More problematic would be if asserted something that is non-trivially inferred by reasoner.

Reply:
@dosumis 

> Delete subClassOf assertion - don't need to duplicate genus also in EquivalentClass assertion. Not a big deal, but I prefer to avoid. More problematic would be if asserted something that is non-trivially inferred by reasoner.

Done, thanks.